src/anshuKJun20/StringPalindrome.java

```diff
@@ -0,0 +1,28 @@
+package anshuKJun20;
+
+public class StringPalindrome {
+	
+		private void finfPalindromeString(String name) {
+			String result = "";
+			for (int index = (name.length() - 1); index >= 0; index--) {
+				result = result + name.charAt(index);
+			}
+			if (name.equalsIgnoreCase(result)) {
+				System.out.println("Given string " + name + " is palindrome");
+			} else {
+				System.out.println("Given string " + name + " is not palindrome");
+			}
+		}
+
+		public void display() {
```

Comment:
You are using display method to call core method.You can use it.But i think in this simple program we don't need this. @KrishnaKTechnocredits  Please correct me if I am wrong.

Reply:
yes Pranita you are right. I have  made changes according to this.

Reply:
Yes correct @PranitaPJun20 we don't need display() method.